### ISSUE TITLE
Please Elaborate on GTK Theme More?

### ISSUE THREAD

**Issue by LoneWolf4713** [kepano/flexoki#80]
Hello, I followed the instructions, restarted two times yet the theme is not getting applied, The way I apply GTK Themes on Hyprland is by installing them in ~/.themes folder and applying them with nwg-look. however your method is different, I may be missing something. Please provide further instructions, or package it like other GTK Themes.

**Comment by kepano** [comment]
@hydroakri do you have any suggestions?

**Comment by hydroakri** [comment]
Rename and overwrite `~/.config/gtk-3.0/gtk.css` and `~/.config/gtk-4.0/gtk.css` with the provided css files.
Some gtk modules require the [adw-gtk](https://github.com/lassekongo83/adw-gtk3) component.

**Comment by kepano** [comment]
@LoneWolf4713 let us know if this works and if you have any suggestions for how to clarify the instructions.

**Comment by hydroakri** [comment]
Wait sorry, I seem to be mistaken about how it works. Themes should be stored in `~/.themes` or `~/.local/share/themes` instead of being replaced manually.
will be fixed later

**Comment by LoneWolf4713** [comment]
@hydroakri exactly, that's how GTK Themes are packaged, I've looked at other themes file and they also seem to include a `index.theme` file, I figure that's how gsettings recognizes these themes, without it, it doesn't. 
have a look at various themes from any site, one such is [gnome-look](https://gnome-look.org) .

**Comment by hydroakri** [comment]
#82 Should be done
